### ISSUE TITLE
what are the pros and cons of ghidra_bridge?

### ISSUE THREAD

**Issue by Jiadosi** [justfoxing/ghidra_bridge#13]
hi,
I am curious what is the difference between your ghidra_bridge and Ghidra's own server?

best

**Comment by justfoxing** [comment]
Entirely different use cases. Ghidra's server is for doing distributed reverse engineering and sharing analyses.

ghidra_bridge, as mentioned in the README, is for enabling you to use python3 as your python scripting environment for Ghidra scripts, instead of being stuck in the python2/jython environment distributed with Ghidra. While it uses some of the same terms, like "server", it has nothing to do with the Ghidra server.

TLDR: if you need to work with someone else while reverse engineering, you're probably looking for the Ghidra server. If you need a modern python environment or to use a fancy new python package in your Ghidra scripts, you're looking for ghidra_bridge.

**Comment by Jiadosi** [comment]
cool
appreciate your work
